src/promptflow/promptflow/_core/tool_meta_generator.py

```diff
@@ -37,8 +37,14 @@ def generate_prompt_tool(name, content, prompt_only=False, source=None):
     try:
         inputs = get_inputs_for_prompt_template(content)
     except Exception as e:
-        msg = f"Parsing jinja got exception: {e}"
-        raise JinjaParsingError(msg) from e
+        error_type_and_message = f"({e.__class__.__name__}) {e}"
+        raise JinjaParsingError(
+            message_format=(
+                "Generate tool meta failed for {tool_type} tool. Jinja parsing failed: {error_type_and_message}"
```

Comment:
Also add name info in error message?
Or that may be hard to find which node needs to fix.

Reply:
Not necessary, all the errors thrown from tool_meta_generator.py will be wrapped as ToolLoadError, which will contain the node name.

Reply:
Checked about call hierarchy about `generate_prompt_tool`, seems that's not true..

Reply:
For flow run scenario, the error thrown will be wrapped as ToolLoadError. For the other scenarios, let‘s deal with them later.